From a3d42317b104f7dc12dd9821293ddd079df352d4 Mon Sep 17 00:00:00 2001
From: Christoph Junghans <junghans@votca.org>
Date: Sat, 12 Aug 2017 10:36:24 -0600
Subject: [PATCH] cmake: require c99 standard

Some files, e.g. encode_accessors.c, use loop initial declarations
that are only allowed in C99 mode. Newer compilers enable that mode
automatically, but olders might fail. CMAKE_C_STANDARD got add in
CMake-3.1, so let's increase the minimal version by 0.1.

Signed-off-by: Christoph Junghans <junghans@votca.org>
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e8b0e93..22f24e0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.0.0)
+cmake_minimum_required(VERSION 3.1)
+set(CMAKE_C_STANDARD 99)
 include(CheckIncludeFiles)
 include(TestBigEndian)
 include(CheckCSourceCompiles)
-- 
2.13.0

